R/set-record-option.R

```diff
@@ -0,0 +1,110 @@
+
+#' Add or modify a record option
+#'
+#' Add a new record option or adjust the value/presence of an existing one.
+#'
+#' Note that only estimation, data, and table records are currently supported.
```

Comment:
With the switch to support "standard" options (i.e. things defined via `*_option_types` and `*_option_names`), you can lift this requirement and allow any record type.